sorc/gaussian_sfcanl.fd/weight_gen/run.theia.sh

```diff
@@ -53,7 +53,7 @@ module load hdf5/1.8.14
 RWG=/scratch4/NCEPDEV/nems/noscrub/emc.nemspara/soft/esmf/7.1.0r/bin/ESMF_RegridWeightGen
```

Comment:
This whole script can just get deleted since it's for Theia...or renamed to keep and run everywhere?

Reply:
It should/could be renamed.
Bigger question is that this script and utility belongs in UFS-utils.  This utility is not used in the workflow.
I will create an issue in the ufs-utils and tag @yangfanglin @GeorgeGayno-NOAA for further action.

Reply:
https://github.com/ufs-community/UFS_UTILS/issues/673